ledger-core/v2/runner/runner.go

```diff
@@ -8,43 +8,41 @@ package runner
 import (
 	"bytes"
 	"context"
-	"fmt"
 	"sync"
 
-	"github.com/google/uuid"
 	"github.com/pkg/errors"
 
 	"github.com/insolar/assured-ledger/ledger-core/v2/insolar"
 	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/payload"
 	"github.com/insolar/assured-ledger/ledger-core/v2/instrumentation/inslogger"
-	"github.com/insolar/assured-ledger/ledger-core/v2/runner/calltype"
+	"github.com/insolar/assured-ledger/ledger-core/v2/runner/callcontext"
 	"github.com/insolar/assured-ledger/ledger-core/v2/runner/execution"
-	"github.com/insolar/assured-ledger/ledger-core/v2/runner/executionupdate"
 	"github.com/insolar/assured-ledger/ledger-core/v2/runner/executor"
 	"github.com/insolar/assured-ledger/ledger-core/v2/runner/executor/builtin"
+	"github.com/insolar/assured-ledger/ledger-core/v2/runner/machinetype"
 	"github.com/insolar/assured-ledger/ledger-core/v2/runner/requestresult"
 	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/throw"
 	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/descriptor"
 )
 
 type Service interface {
-	ExecutionStart(ctx context.Context, execution execution.Context) (*executionupdate.ContractExecutionStateUpdate, uuid.UUID, error)
-	ExecutionContinue(ctx context.Context, id uuid.UUID, result []byte) (*executionupdate.ContractExecutionStateUpdate, error)
-	ExecutionAbort(ctx context.Context, id uuid.UUID)
-	ExecutionClassify(ctx context.Context, execution execution.Context) calltype.ContractCallType
```

Comment:
Classify will be needed

Reply:
@sexyhamster will reintroduce it in his patch

Reply:
as we've discussed, please put it back :)

Reply:
In next PR, if you don't mind.

Reply:
I'm ok